telemetry/specs/telemetry_report.mdk

```diff
@@ -473,48 +548,38 @@ particular deployment.
 
     See the INT specification [^INT] for definitions of the remaining metadata.
+    
+* Domain Specific ID (16b)
+  : The unique ID of the INT Domain
+
+* DSMdbit (16b): Domain Specific Md Bits
+  : Bitmap that indicates which optional domain specific metadata is 
+    present in the record of the telemetry report. Each bit represents 
+    4 octets of domain specific optional metadata.
+    
+* DSMdstatus (16b): Domain Specific Md Status
+  : Bitmap that indicates the domain specific metadata status. 
+  
+* Variable Optional Baseline Metadata
+  : The metadata corresponding to the RepMdBits, 4 octets for each bit, except 8 octets 
+    for bits 4, 5 & 6.
+    
+* Variable Optional Domain Specific Metadata
+  : The metadata corresponding to the DSMdBits, 4 octets for each bit.
+    
+* Additional DS Extension Data
+  : Additional Domain Specific Specific Extension Data not associated with DSMdbit.
+  
+* Truncated Packet
+  : L2/L3/ESP/L4 of the packet for flow details 
 
-* D: Dropped
-  : Indicates that at least one packet matching a drop watchlist was dropped.
-
-* Q: Congested Queue Association
-  : Indicates the presence of congestion on a monitored queue.
-
-* F: Tracked Flow Association
-  : Indicates that this telemetry report is for a tracked flow, i.e. the
-    packet matched a flow watchlist at Source device (in case of INT-MX, INT-MD
-    or IOAM) or
-    at individual devices (in case of INT-XD). The report might include INT or IOAM
-    metadata beyond the inner ethernet header. Other telemetry reports are
-    likely to be received for the same tracked flow, from the same network
-    device and (in case of drop reports, individual reports, or path changes)
-    from other network devices.
-
-* hw_id
-  : Identifies the hardware subsystem within the network device that generated
-    this report. For example, in a chassis with multiple linecards this could
-    identify a specific linecard, or a subsystem within a linecard. The hw_id
-    is unique within the scope of a Switch id.
-
-* Switch id
-  : The unique ID of a switch (generally administratively assigned). Switch IDs
-    must be unique within a management domain.
-
-* Sequence Number
-  : Reflects the sequence of reports from a specific combination of
-    (Switch id, hw_id) to a particular telemetry report destination. This
-    can be used to detect loss of telemetry reports before they reach their
-    intended destination.
-
-* Ingress Timestamp
-  : The device local time when the packet was first received on the ingress
-    physical or logical port, in nanoseconds.
 
 ## Flow Identification
 
 There is no explicit metadata defined for flow identification. The
 expectation is that packet headers will be included in the telemetry
-report, allowing the distributed telemetry monitoring system to categorize
+report or domain specific flow identification metadata will be included, 
```

Comment:
nit: This is a little hard to parse. How about:

The expectation is that either:

- a truncated packet fragment including packet headers will be included in the telemetry report,
  allowing the distributed telemetry monitoring system to categorize and identify flows in any
  manner that it desires, or

- domain specific flow identification metadata will be included.

Reply:
Agree. Modified the text.